test/bigtest/tests/package-usage-consolidation-test.js

```diff
@@ -92,7 +92,8 @@ describe('PackageShowUsageConsolidation', () => {
           await PackageShowPage.usageConsolidation.content.whenLoaded();
         });
 
-        describe('waiting for axe to run', () => {
+        // Try to run test after STCOM-788
```

Comment:
Please add a TODO prefix:
```
// TODO: Try to run test after STCOM-788
```
We have this format for other tests that have to be enabled after some action